src/scope.ml

```diff
@@ -84,11 +84,11 @@ let find_sym : prt:bool -> prv:bool -> bool -> sig_state -> qident ->
   in
   begin
     match (prt, prv, s.sym_expo) with
-    | (false, _    , Protected) ->
+    | (false, _    , Protec) ->
         if s.sym_path <> st.signature.sign_path then
           (* Fail if symbol is not defined in the current module. *)
           fatal pos "Protected symbol not allowed here."
-    | (_    , false, Private  ) ->
+    | (_    , false, Privat) ->
         fatal pos "Private symbol not allowed here."
     | _                         -> ()
```

Comment:
The arrow is not aligned.

Reply:
Fixed in d7b99a6f1252f9d51d90833cc70c9ff9774a01b8.